crypto/dilithium/pqcrystals_dilithium_ref_common/sign.c

```diff
@@ -63,73 +68,92 @@ int crypto_sign_keypair(ml_dsa_params *params, uint8_t *pk, uint8_t *sk) {
   /* Compute H(rho, t1) and write secret key */
   shake256(tr, TRBYTES, pk, params->public_key_bytes);
   pack_sk(params, sk, rho, tr, key, &t0, &s1, &s2);
-
   return 0;
 }
 
 /*************************************************
-* Name:        crypto_sign_signature
+* Name:        crypto_sign_keypair
 *
-* Description: Computes signature.
+* Description: FIPS 204: Algorithm 1 ML-DSA.KeyGen
+*              Generates public and private key.
 *
 * Arguments:   - ml_dsa_params: parameter struct
+*              - uint8_t *pk: pointer to output public key (allocated
+*                             array of CRYPTO_PUBLICKEYBYTES bytes)
+*              - uint8_t *sk: pointer to output private key (allocated
+*                             array of CRYPTO_SECRETKEYBYTES bytes)
+*
+* Returns 0 (success) -1 on failure
+**************************************************/
+int crypto_sign_keypair(ml_dsa_params *params, uint8_t *pk, uint8_t *sk) {
+  uint8_t seed[SEEDBYTES];
+  /* Get randomness for rho, rhoprime and key */
```

Comment:
```suggestion
```

Reply:
removed in [061ebce](https://github.com/aws/aws-lc/pull/1999/commits/061ebceca90a012a0e8ba261f71c8724251a42ea)